dockers/app/requirements.pip3.quefunciona_antics

```diff
@@ -0,0 +1,11 @@
+APScheduler==3.3.1
+Flask-SocketIO==2.8.6
+iniparse==0.4
+rethinkdb==2.3.0.post6
+pynpm==0.1.1
+graphyte==1.4
+pem==18.2.0
+Flask-Login==0.4.1
+xmltodict==0.11.0
+python-engineio==3.8.1
+python-socketio==4.1.0
```

Comment:
Is this file necessary? It could be fetched through git history

Reply:
test for upgrading